InsertFunctions.java

```diff
@@ -0,0 +1,79 @@
+
+import java.sql.*;
+
+
+public class InsertFunctions {
+/*Java class to access and insert new data into the DataBase
+ * for the modules, categories, courses, and streams tables.
+ */
+	
+	
+	 
+	public static void main(String[] args){
+		try {
+			Statement state;
+			Connection c1;
+			Class.forName("oracle.jdbc.driver.OracleDriver");
+			c1 = DriverManager.getConnection("jdbc:oracle:thin:@localhost:1521:XE", "Student_Performance", "Student_Performance");
+			state = c1.createStatement();
+		} catch (Exception e) {
+			// TODO Auto-generated catch block
+			e.printStackTrace();
+		}
+	}
+	
+	
+		
+		
+		//INSERT FOR STREAM
+		public void insertStream(Statement state, String id, String name){
+		try {
+		int r1 =state.executeUpdate("INSERT INTO stream VALUES('"+id+"', '"+name+"')");
+		System.out.println(r1+" rows inserted."); //Take this line out if you don't want a confirmation message
+		
+		}
+		catch (Exception e) {
+			// TODO Auto-generated catch block
+			e.printStackTrace();
+		}
+		}
+		//INSERT FOR CLASS
+		
+		public static void insertClass(Statement state, String id, String name, String idFK){

```

Comment:
LOL!!!!!!!! Probably shouldn't leave that parameter name in for real though. 😆 🤣